flax/nnx/nnx/bridge/wrappers.py

```diff
@@ -225,14 +230,16 @@ def update_variables(self, module):
     types = set(jax.tree.leaves(
       jax.tree.map(lambda x: x.type, state,
                     is_leaf=lambda x: isinstance(x, nnx.VariableState))))
-    types = variableslib.sort_variable_types(types)
+    types = bv.sort_variable_types(types)
```

Comment:
`sort_variable_types` expects type `list` but we can change the type to `Iterable`

Reply:
Thanks for the catch! Done.